components/VM_Arm/configurations/vm.h

```diff
@@ -68,13 +68,36 @@
     attribute int num_extra_frame_caps; \
     attribute int extra_frame_map_address; \
     attribute { \
-        string linux_ram_base; \
```

Comment:
Since we keep the `linux_address_config` block, could you change this commit to not delete this here, but keep it and add the new `vm_address_config` block below? That makes the actual chances a bit easier to track.

Reply:
The issue was a couple of `=""` values that made the change harder to track. Its fixed now.